.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+
+  1. Loaded core with llnode node/location -c core/location
+  
+  2. Ran command v8 '...'
+  
+  3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Console output**
+If applicable, add console output to help explain your problem.
+
+**Local environment (please complete the following information):**
+ - Version of llnode [e.g. 4.0.0]
+ 
+   Obtained using `npm view llnode version`
+ - OS Info: [e.g. Linux ub 5.15.0-46-generic #49~20.04.1-Ubuntu SMP Thu Aug 4 19:15:44 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux] 
+ 
+   Obtained using `uname -a` or on Windows get the important information with `wmic os get OsArchitecture && wmic os get name`  
+ - lldb Version [e.g. 14.0.6] 
+   
+   Obtained using `lldb --version`
+ - Version of node.js [e.g. 16.17.0] 
+   
+   Obtained using `node -v`
+
+**Server where the core was dumped (please complete the following information):**
+ - OS Info: [e.g. Linux ub 5.15.0-46-generic #49~20.04.1-Ubuntu SMP Thu Aug 4 19:15:44 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux] 
+ 
+   Obtained using `uname -a` or on Windows get the important information with `wmic os get OsArchitecture && wmic os get name`
+ - lldb Version [e.g. 14.0.6] 
+ 
+   Obtained using `lldb --version`
+ - Version of node.js [e.g. 16.17.0] 
+   
+   Obtained using `node -v`
```

Comment:
```suggestion
**Server where the core was dumped (please run the following commands):**
- Linux or macOS: `uname -a && lldb -v && node -v`
- or on Windows: `wmic os get OsArchitecture && wmic os get name && lldb -v && node -v`
```

Reply:
Not to sure about this. If a command fails it can be hard to discern what's going on and would add to frustration also we want the out put to be explicit as lldb,node and llnode have overlapping versions numbers.

Reply:
Understood.  In that case, use `-v` or `--version` consistently, and let's land your improvements as they are.